db/fixtures/customization_templates.yml

```diff
@@ -274,3 +274,10 @@
     />\n</unattend>"
   :type: CustomizationTemplateSysprep
   :system: true
+
+- :name: Basic root pass template
+  :description: This template takes use of rootpassword defined in the UI
+  :script: "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False"
+  :type: CustomizationTemplateCloudInit
+  :pxe_image_type_id: 1
```

Comment:
OK makes sense, how I can tackle it? (and why is the pxe_image_type_id required?)
I can do something like this instead of this db:seed
```
def add_basic_root_template
    unless CustomizationTemplate.find_by_name("Basic root pass template")
      options = { :name => "Basic root pass template",
                  :description => "This template takes use of rootpassword defined in the UI",
                  :script => "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False",
                  :type => "CustomizationTemplateCloudInit",
                  :pxe_image_type_id => PxeImageType.first.id }
      CustomizationTemplate.new(options).save
    end
  end
```
but then it will be available only for users who used deployment at least once.